examples/generate_image_dataset/generate_images.py

```diff
@@ -134,11 +134,11 @@ def single_step(self, action=None, get_observation=False):
 
                     # discard normals that aren't properly normalized, i.e., length of 1.0
                     discard_mask = np.logical_not(np.isclose(np.linalg.norm(obs_render_pass_vis, axis=2), 1.0, rtol=0.001, atol=0.001))
-                    obs_render_pass_vis = np.clip(0.0, 1.0, (obs_render_pass_vis + 1.0) / 2.0)
+                    obs_render_pass_vis = np.clip((obs_render_pass_vis + 1.0) / 2.0, 0.0, 1.0)
```

Comment:
This same bug (introduced by me, my bad 😅) also occurs a few lines above for the `depth` pass.

Reply:
Updated.